src/main/resources/assets/enhancedlootbags/lang/en_US.lang

```diff
@@ -17,4 +19,64 @@ enhancedlootbags.nei.recipe.weight=Weight: %s
+enhancedlootbags.group.steam_age=Steam Age
+enhancedlootbags.group.ulv=ULV
+enhancedlootbags.group.tier_1_lv=Tier 1 LV
+enhancedlootbags.group.tier_2_mv=Tier 2 MV
+enhancedlootbags.group.tier_3_hv=Tier 3 HV
+enhancedlootbags.group.tier_4_ev=Tier 4 EV
+enhancedlootbags.group.tier_5_iv=Tier 5 IV
+enhancedlootbags.group.magic_novice=Magic Novice
+enhancedlootbags.group.magic_adept=Magic Adept
+enhancedlootbags.group.magic_master=Magic Master
+enhancedlootbags.group.magic_grandmaster=Magic Grandmaster
+enhancedlootbags.group.magic_grandmaster_-_unique=Magic Grandmaster - Unique
+enhancedlootbags.group.legendary=Legendary
+enhancedlootbags.group.legendary_-_unique=Legendary - Unique
+enhancedlootbags.group.bm_novice=BM Novice
+enhancedlootbags.group.bm_adept=BM Adept
+enhancedlootbags.group.bm_master=BM Master
+enhancedlootbags.group.forest_ranger_basic=Forest Ranger Basic
+enhancedlootbags.group.forest_ranger_advanced=Forest Ranger Advanced
+enhancedlootbags.group.forest_ranger_expert=Forest Ranger Expert
+enhancedlootbags.group.space_invaders_moon=Space Invaders Moon
+enhancedlootbags.group.space_invaders_mars=Space Invaders Mars
+enhancedlootbags.group.space_invaders_asteroids=Space Invaders Asteroids
+enhancedlootbags.group.bees_basic=Bees Basic
+enhancedlootbags.group.bees_advanced=Bees Advanced
+enhancedlootbags.group.bees_expert=Bees Expert
+enhancedlootbags.group.fast_food=Fast Food
+enhancedlootbags.group.slow_food=Slow Food
+enhancedlootbags.group.haute_cuisine=Haute Cuisine
+enhancedlootbags.group.dessert=Dessert
+enhancedlootbags.group.transportation_3_class=Transportation 3 Class
+enhancedlootbags.group.transportation_2_class=Transportation 2 Class
+enhancedlootbags.group.transportation_1_class=Transportation 1 Class
+enhancedlootbags.group.seeds=Seeds
+enhancedlootbags.group.gardens=Gardens
+enhancedlootbags.group.ae_basic=AE Basic
+enhancedlootbags.group.ae_good=AE Good
+enhancedlootbags.group.ae_advanced=AE Advanced
+enhancedlootbags.group.ae_master=AE Master
+enhancedlootbags.group.tier_6_luv=Tier 6 LuV
+enhancedlootbags.group.tier_7_zpm=Tier 7 ZPM
+enhancedlootbags.group.tier_8_uv=Tier 8 UV
+enhancedlootbags.group.tier_9_uhv=Tier 9 UHV
+enhancedlootbags.group.computer_floppies=Computer Floppies
+enhancedlootbags.group.tier_1_computers=Tier 1 Computers
+enhancedlootbags.group.tier_2_computers=Tier 2 Computers
+enhancedlootbags.group.tier_3_computers=Tier 3 Computers
+enhancedlootbags.group.tier_3.5_computers=Tier 3.5 Computers
+enhancedlootbags.group.tier_4_computers=Tier 4 Computers
+enhancedlootbags.group.ic2_crops=IC2 Crops
+enhancedlootbags.group.witchery_novice=Witchery Novice
+enhancedlootbags.group.witchery_adept=Witchery Adept
+enhancedlootbags.group.witchery_master=Witchery Master
+enhancedlootbags.group.witchery_witch=Witchery Witch
+enhancedlootbags.group.witchery_deamon=Witchery Deamon
+enhancedlootbags.group.hee-basic=HEE-Basic
+enhancedlootbags.group.hee-advanced=HEE-Advanced
```

Comment:
These lootbags are not defined in this mod so their local names should not be part of the lootbags mod.